### PR TITLE
fix(backend): ensure cookie expiry date is extended when too old

### PR DESCRIPTION
**Pull request by grignards**
See #3431 

## Checkpoints

These points must be checked before merging. Please don't edit them out.

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

Testing method: start the backend with a small cookie expiry date in `development.ini` (`session.cookie_expires = 30` for example). Then play with Tracim and check in the dev toobar that the cookie expiry date is extended properly.

### CODE REVIEW

File: backend/tracim_backend/lib/utils/authentification.py [hunk 1]
```diff
@@ -143,9 +148,35 @@ def get_current_user(self, request: TracimRequest) -> typing.Optional[User]:
             user = self._get_user_api(request).get_one(user_id=request.unauthenticated_userid)
         except UserDoesNotExist:
             user = None
-        # do not allow invalid_user + ask for cleanup of session cookie
+        # do not allow invalid_user
         if not user or not user.is_active or user.is_deleted:
             return None
+
+        # ensure cookie expiry date is updated if it is too old
```
Comment (inkhey) [inline backend/tracim_backend/lib/utils/authentification.py:155]: Follow coding rule for comment here : 
```suggestion
        # INFO - SG - 2020-08-13  ensure cookie expiry date is updated if it is too old
```

Comment (raphj) [reply backend/tracim_backend/lib/utils/authentification.py:155]: I feel like we should re-discuss this rule. It seems a bit overkill in most cases.

Comment (grignards) [reply backend/tracim_backend/lib/utils/authentification.py:155]: I agree: for "simple" comments it seems too much to me. I would reserve the `INFO` when explaining a "strange"/"unexpected" behavior.

**Comment by gbonaspetti** [comment]
Tested on branch.

**Pull request merged**